Source/PyFR/Gradients.h

```diff
@@ -397,11 +401,11 @@ class Gradients
 
     //We need to compute the gradient of all the data.
     vtkm::cont::CoordinateSystem coordField = input.GetCoordinateSystem();
-    vtkm::cont::Field densityField = input.GetField(PyFRData::FieldName(0));
-    vtkm::cont::Field pressureField = input.GetField(PyFRData::FieldName(1));
-    vtkm::cont::Field veluField = input.GetField(PyFRData::FieldName(2));
-    vtkm::cont::Field velvField = input.GetField(PyFRData::FieldName(3));
-    vtkm::cont::Field velwField = input.GetField(PyFRData::FieldName(4));
+    vtkm::cont::Field densityField = input.GetField("density");
+    vtkm::cont::Field pressureField = input.GetField("pressure");
```

Comment:
Yes, one of the quirks of the current in-situ pipeline is that it works in terms of the conservative variables (rho, rho*u, rho*v, rho*w, E) whereas the VTK export code in PyFR performs the conversion back to primitive variables as (rho, u, v, w, p).  The pipeline currently takes are of diving through by rho to get the velocities although we still are left with the total energy rather than the pressure.